bin/OSPData/adera/ph_engine.sturdy.gltf

```diff
@@ -68,7 +68,7 @@
                         "type" : "FreeEnergyGenerator"
                     }
                 ],
-                "massdry" : 300,
+                "massdry" : 1000,
```

Comment:
Annoyingly github doesn't allow me to comment on a whole file at a time.

is bin/OSPData/adera/ph_rcs.sturdy.bin  being deleted?

Reply:
No, why do you ask?

Reply:
Because github says that 98% of the file is being deleted?

Why are .bin files checked into git? What do they represent?

Reply:
These are bin files containing mostly mesh data. Perhaps its time to use LFS or similar solution?

Reply:
I suspect the 98% is meaning that 98% is remaining, as counterintuitive as it is. It says -256 bytes, and it's a 5kB file.

Reply:
IMO git LFS is half-baked

For things under a megabyte, its not worth it.